src/Util/GeometryUtil/GeometryUtil.js

```diff
@@ -0,0 +1,292 @@
+   */
+  static addBuffer (geometry, radius = 0, projection = 'EPSG:3857') {
+    if (radius === 0) {
+      return geometry;
+    }
+    const geoJsonFormat = new OlFormatGeoJSON({
+      dataProjection: 'EPSG:4326',
+      featureProjection: projection
+    });
+    const geoJson = geometry instanceof OlFeature
+      ? geoJsonFormat.writeFeatureObject(geometry)
+      : geoJsonFormat.writeGeometryObject(geometry);
+    const buffered = buffer(geoJson, radius, {
+      units: 'meters'
+    });
+    if (geometry instanceof OlFeature) {
+      return geoJsonFormat.readFeature(buffered);
+    } else {
+      return geoJsonFormat.readGeometry(buffered.geometry);
+    }
+  }
+
+  /**
+   * Merges multiple geometries into one MultiGeometry.
+   *
+   * @param {ol.geom.Geometry[]} geometries An array of ol.geom.geometries;
+   * @return {ol.geom.Multipoint|ol.geom.MultiPolygon|ol.geom.MultiLinestring} A Multigeometry.
+   */
+  static mergeGeometries(geometries) {
+    const geomType = geometries[0].getType();
+    let mixedGeometryTypes = false;
+    geometries.forEach(geometry => {
+      if (geomType !== geometry.getType()) {
+        mixedGeometryTypes = true;
+      }
+    });
+    if (mixedGeometryTypes) {
+      // Logger.warn('Can not merge mixed geometries into one multigeometry.');
+      return undefined;
+    }
+    let multiGeom;
+    let append;
+    switch (geomType) {
+      case 'Polygon':
+        multiGeom = new OlGeomMultiPolygon();
+        append = multiGeom.appendPolygon.bind(multiGeom);
+        break;
+      case 'Point':
+        multiGeom = new OlGeomMultiPoint();
+        append = multiGeom.appendPoint.bind(multiGeom);
+        break;
+      case 'LineString':
+        multiGeom = new OlGeomMultiLineString();
+        append = multiGeom.appendLineString.bind(multiGeom);
+        break;
+      default:
+        return undefined;
+    }
+    geometries.forEach(geom => append(geom));
+    return multiGeom;
+  }
+
+  /**
+   * Takes two or more polygons and returns a combined polygon.
+   *
+   * @param {ol.geom.Geometry[] | ol.Feature[]} polygons An array of ol.Feature
+   *  or ol.geom.Geomerty instances of type polygon.
+   * @param {String} projection A projection as EPSG code.
+   * @returns {ol.geom.Geometry | ol.Feature} A Feature or Geometry with the
+   * combined area of the polygons.
+   */
+  static union(polygons, projection = 'EPSG:3857') {
+    const geoJsonFormat = new OlFormatGeoJSON({
+      dataProjection: 'EPSG:4326',
+      featureProjection: projection
+    });
+    let invalid = false;
+    const geoJsonsFeatures = polygons.map((geometry) => {
+      const feature = geometry instanceof OlFeature
+        ? geometry
+        : new OlFeature({geometry});
+      if (feature.getGeometry().getType() !== 'Polygon') {
+        invalid = true;
+      }
+      return geoJsonFormat.writeFeatureObject(feature);
+    });
+    if (invalid) {
+      // Logger.warn('Can only create union of polygons.');
```

Comment:
hmm. why not warn? and why don't we iterate over the single parts of the multipolygon in that case?

Reply:
Are holes in (Multi)polygons also covered here?

Reply:
@marcjansen I don't think we do log anywhere in react-geo... We just have `loglevel` as dependency. :see_no_evil: 

@ahennr Yes. I won't add anymore test as these would be more or less just tests agains `turf.js`. But it tested it once in a sandbox.